### DISCUSSION TITLE
Proxy Connections

### DISCUSSION THREAD

**Discussion by tybruno** [ParallelSSH/ssh2-python#151] [Q&A] [answered]
Hello,

How do I setup a proxy connection with ssh2-python? I looked at the documentation but I didn't see any info on it.

Thanks,
Tyler

**Comment by pkittenis** [comment, marked as answer]
Thanks for the interest.

See [parallel-ssh ](https://parallel-ssh.readthedocs.io/en/latest/advanced.html#proxy-hosts-and-tunneling) for a high level client that supports proxying.
